database-handler.py

```diff
@@ -62,4 +62,13 @@ def delete_user(usr_id: str) -> None:
 
     close(db, cursor)
     
-# --- Data --- #
\ No newline at end of file
+# --- Data --- #
+
+def insert_employee(fname: str, sname: str, email: str, birthdate: str) -> bool:
+    pass
+
+def update_employee(fname: str, sname: str, email: str, birthdate: str) -> bool:
+    pass
+
+def delete_employee(fname: str, sname: str, email: str, birthdate: str) -> bool:
+    pass
```

Comment:
This may not need to return a bool, as it is only possible to delete if the employee is present in the table.